### PR TITLE
Input hoc - update label proptype

### LINKED ISSUE #455: Select - set PropType.node to validate 'label' prop
We need to change the type checking of the `label` prop to allow render `PropType.node`, to avoid warnings if we need to render anything else than `string`, like the example below in webapp. 

captura de tela de 2018-07-27 16-35-12

captura de tela de 2018-07-27 16-35-46

### PR DESCRIPTION
**Pull request by fgameiro**
Updates `label` proptype validation of `Input HOC` component to be used on `Select Input` component.

## Closes Issues

* Closes #455 

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

**Comment by fgameiro** [comment]
@fczuardi currently, the more info is the only use case for this feature...
I think I'll go with the `afterLabel` (or maybe `moreInfoText`?) option then.

**Comment by fgameiro** [comment]
@fczuardi please, take a look at the new commits :)

File: packages/hoc-input/src/formInput.js [hunk 1]
```diff
@@ -28,7 +29,8 @@ export const formInputProps = {
   refName: PropTypes.string,
   clearable: PropTypes.bool,
   clearIcon: PropTypes.string,
-  leftIcon: PropTypes.string
+  leftIcon: PropTypes.string,
+  moreInfoContent: PropTypes.node
```
Comment (fczuardi) [inline packages/hoc-input/src/formInput.js]: better :+1:

File: packages/hoc-input/src/styledComponents/Wrapper.js [hunk 2]
```diff
@@ -130,6 +130,7 @@ const Wrapper = styled.div`
     letter-spacing: 2px;
     margin-bottom: 8px;
     color: ${labelColor};
+    display: inline-block;
```
[the remark this answers is not in the document]
Comment (fczuardi) [reply packages/hoc-input/src/styledComponents/Wrapper.js]: maybe changing the markup instead, by putting the more info as a child of Text...

[the remark this answers is not in the document]
Comment (fczuardi) [reply packages/hoc-input/src/styledComponents/Wrapper.js]: yeah, nevermind the idea of moving to inside text, that will apply uppercase on more info

File: packages/hoc-input/src/formInput.js [hunk 3]
```diff
@@ -138,17 +142,24 @@ const formInput = WrappedComponent => {
...
+            <Text
+              title
+              className="label"
+              size={fontSizes.SMALL}
+              lineHeight={1.14}
+            >
+              {label}
+            </Text>
+            {moreInfoContent && (
+              <MoreInfo className="moreInfo" tooltipAlign={alignments.LEFT}>
```
Comment (alcferreira) [inline packages/hoc-input/src/formInput.js]: We should receive an align property and pass it to `MoreInfo` component.

**Pull request merged**